www/include/eventLogs/xml/data.php

```diff
@@ -113,6 +113,10 @@
     'search_service' => FILTER_SANITIZE_STRING,
     'export' => FILTER_SANITIZE_STRING,
 );
+
```

Comment:
please use FILTER_VALIDATE_INt instead of FILTER_SANITIZE_NUMBER_INT. As an integer is expected and chars should not be used for these params : eg : $id = "12;3", will result in $id == 123, when it should result in $id == false

Reply:
lines 90, 91 & 96

Reply:
Done